docs/intro/install.md

```diff
@@ -39,24 +37,6 @@ Then run this command to move the `starport` executable to `/usr/local/bin/`:
 sudo mv starport /usr/local/bin/
 ```
 
-### Error while running command `protoc`
```

Comment:
bundling protoc might have fixed this pain point? can we remove this protoc troubleshooting? https://github.com/tendermint/starport/blob/develop/docs/intro/install.md#error-while-running-command-protoc  
(bundling protoc was a fabulous pain reliever)

Reply:
Yes, we can!